client/js/components/GameView/GameDialogs.vue

```diff
@@ -0,0 +1,193 @@
+      :target="game.oneOffTarget"
+      :twos-in-hand="twosInHand"
+      :twos-played="twosPlayed"
+      @resolve="resolve"
+      @counter="counter($event)"
+    />
+    <cannot-counter-dialog
+      v-model="showCannotCounterDialog"
+      :one-off="game.oneOff"
+      :opponent-queen-count="opponentQueenCount"
+      :player-two-count="playerTwoCount"
+      :twos-played="twosPlayed"
+      :target="game.oneOffTarget"
+      @resolve="resolve"
+    />
+    <four-dialog v-model="discarding" @discard="discard" />
+    <three-dialog
+      v-model="pickingFromScrap"
+      :one-off="game.oneOff"
+      :scrap="scrap"
+      @resolveThree="resolveThree($event)"
+    />
+    <seven-double-jacks-dialog
+      v-model="showSevenDoubleJacksDialog"
+      :top-card="topCard"
+      :second-card="secondCard"
+      @resolveSevenDoubleJacks="resolveSevenDoubleJacks($event)"
+    />
+    <game-over-dialog v-model="gameIsOver" :player-wins="playerWins" :stalemate="stalemate" />
+    <reauthenticate-dialog v-model="mustReauthenticate" />
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex';
+
+import CannotCounterDialog from '@/components/GameView/CannotCounterDialog.vue';
+import CounterDialog from '@/components/GameView/CounterDialog.vue';
+import FourDialog from '@/components/GameView/FourDialog.vue';
+import GameOverDialog from '@/components/GameView/GameOverDialog.vue';
+import ReauthenticateDialog from '@/components/GameView/ReauthenticateDialog.vue';
+import SevenDoubleJacksDialog from '@/components/GameView/SevenDoubleJacksDialog.vue';
+import ThreeDialog from '@/components/GameView/ThreeDialog.vue';
+
+export default {
+  name: 'GameDialogs',
+  components: {
+    CannotCounterDialog,
+    CounterDialog,
+    FourDialog,
+    GameOverDialog,
+    ReauthenticateDialog,
+    SevenDoubleJacksDialog,
+    ThreeDialog,
+  },
+  props: {
+    resolvingSeven: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  computed: {
+    ...mapGetters([
+      'discarding',
+      'player',
+      'opponent',
+      'opponentPointTotal',
+      'opponentQueenCount',
+      'resolvingSeven',
+    ]),
+    game() {
+      // TODO: Figure out a better way to do this, mapping the whole module is a
+      // bit unusual-- the usages can probably be changed to only need a subset,
+      // or moved to some sort of store method or getter
+      return this.$store.state.game;
+    },
+    gameIsOver() {
+      return this.$store.state.game.gameIsOver;
+    },
+    hasTwoInHand() {
+      return this.twosInHand.length > 0;
+    },
+    myTurnToCounter() {
+      return this.game.myTurnToCounter;
+    },
+    // TODO: Refactor and combine usage in GameView.vue
+    mustReauthenticate: {
+      get() {
+        return this.$store.state.auth.mustReauthenticate;
+      },
+      set(val) {
+        this.$store.commit('setMustReauthenticate', val);
+      },
+    },
+    pickingFromScrap() {
+      return this.$store.state.game.pickingFromScrap;
```

Comment:
All the references to `this.$store.state.game` could be replaced with `this.game`